### PR TITLE
Fixed: send of empty messages

### PR DESCRIPTION
**Pull request by brenoxp**
@RocketChat/iOS

### CODE REVIEW

**Comment by rafaelks** [comment]
@brenoxp Hey, thanks for your PR! But, how were you able to send a message with 0 chars? The "Send" button should be disabled and the keyboard "Send" button too...

**Comment by brenoxp** [comment]
Yes, you're right. But I was testing the app, and I was able to send empty messages on simulator pressing enter on Mac keyboard. I think it's never too much to have twice verification.

**Comment by rafaelks** [comment]
Got it, makes sense.

Can you just change it to check actually the `message` value after the `guard`? And make the `if` statement not single line.

Thank you!

File: Rocket.Chat/Controllers/Chat/ChatViewController.swift [hunk 1]
```diff
@@ -249,6 +249,11 @@ final class ChatViewController: SLKTextViewController {
 
     fileprivate func sendMessage() {
         guard let message = textView.text else { return }
+        
+        if textView.text.characters.count == 0 {
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Chat/ChatViewController.swift]: You can use `message` instead of `textView.text`. Can you change it, please?

**Comment by rafaelks** [comment]
Thank you @brenoxp!

**Comment by brenoxp** [comment]
You're welcome :)

**Pull request merged**